MekHQ/src/mekhq/campaign/Campaign.java

```diff
@@ -4847,37 +4842,38 @@ public TargetRoll getTargetFor(IPartWork partWork, Person tech) {
             target.addModifier(getFaction().getEraMod(getGameYear()), "era");
         }
 
-        boolean isOvertime = false;
+        final boolean isOvertime;
```

Comment:
What was wrong with the way it was before? Initializing a variable to its default state is a perfectly valid coding pattern.

Reply:
This is a way to handle it that means you know the value is set here and never changes. I find it easier when debugging, as I don't have to trace down any other uses.